src/mmrelay/plugin_loader.py

```
@@ -798,6 +843,318 @@ def _refresh_dependency_paths() -> None:
+            )
+            marker_file.write("\n")
+            marker_file.flush()
+            os.fsync(marker_file.fileno())
+            tmp_path = marker_file.name
+        os.replace(tmp_path, marker_path)
+    finally:
+        if tmp_path and os.path.exists(tmp_path):
+            try:
+                os.unlink(tmp_path)
+            except OSError:
+                pass
```

Ruff (0.15.11)

[warning] Use `contextlib.suppress(OSError)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(OSError): ...`

(SIM105)